content/en/blog/8-must-know-serverless-terms.md

```diff
@@ -0,0 +1,158 @@
++++
+title = "8 Must Know Serverless Terms For A Developer"
+date = "2022-03-01T11:30:34+05:30"
+author = "Atulpriya Sharma"
+description = "Planning to start your cloud native journey? Here are 8 Must Know Serverless Terms For A Developer."
+categories = ["General"]
+type = "blog"
+images = ["/images/featured/serverless-terms.png"]
++++
+
+I’ve been a mobile developer for most of my professional career.
+The transition from Java to Android wasn’t a big deal. However, over the years the entire technological landscape has transformed, thanks to serverless and microservices. Many of the mobile apps that I developed have a serverless component running somewhere on the cloud.
+
+What that meant is that a lot of us developers had to get a hang of the serverless world.
+So to help all of you looking to get started with the serverless journey, here are the **8 Must Know Serverless Terms** you should know as a developer.
+
+{{< figure src="/images/featured/serverless-terms.png" alt="Planning to start your cloud native journey? Here are 8 Must Know Serverless Terms For A Developer." height="600" width="1000">}}
+
+## Serverless Terms You Should Know
+
+Serverless is the present and future of the software industry.
```

Comment:
This is a bold statement. 

We can say instead

Serverless is popular amongst developers for ease of building and deployment.

Reply:
Understood, done.